howto-website.md

```diff
@@ -43,7 +45,7 @@ In the *front-matter* (but not in the text itself), you need to replace any `&`
 The recommended way to host a collaborative note book, e.g. for taking meeting minutes (live notes),
 is to use a collaborative editing tool utilising Markdown directly.  This makes it trivial to move the content into the HSF website for archiving.
 
-[CodiMD](https://hackmd.io/c/codimd-documentation/%2Fs%2Fcodimd-documentation) is the suggested choice as it has been designed for collaboraitve editing of Markdown files. Unfortunately, the [Hackmd](https://hackmd.io) free service is now restricted to 4 editors. Another possibility, if you have a CERN account, is to use CERNBox which makes CodiMD available to edit Markdown files: you can then define a public link to the document (similar to Google Docs public links) to allow those without a CERN account to edit the file.
+If you have a CERN account, is to use CERNBox which makes CodiMD available to edit Markdown files: you can then define a public link to the document (similar to Google Docs public links) to allow those without a CERN account to edit the file. Similarly, the Helmholtz Association has a web based markdown instance that can be used at <https://notes.desy.de>.
```

Comment:
Hey, some of the words "is to use CERNBox which makes" do not read well. Are there any missing words or words too many?

Reply:
Probably, based on the previous version, `another possibility` disappeared...

Reply:
You are absolutely right, that sentence was a disaster! I revised it and it should read better now.

Reply:
Thanks. I capitalised to Markdown for consistency. Good to go :-).